feature-libs/product-configurator/rulebased/occ/cpq/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './converters/index';
+export * from './cpq-configurator-occ.module';
+export * from './cpq-configurator-occ.service';
```

Comment:
All those files makes sense only when CPQ backend is in place. `CpqConfiguratorOccService` is a dependency of `CpqConfiguratorRestAdapter` which lives in `rulebased/cpq` folder.

Does it make sense for you to move all those files entirely from folder `rulebased/occ/cpq` to `rulebased/cpq/**occ**`?

The advantage would be that all logic related to the CPQ integration lives in one main folder  `/rulebased/cpq/...`.

Reply:
Picked

Reply:
Done